### PR TITLE
Fix broken specs

### PR DESCRIPTION
**Pull request by pbollenbeck**
Specs have been failing for a while. These changes fix the broken specs and introduce a travis build status badge in the readme.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,12 +6,10 @@ cache: bundler
 language: ruby
 rvm:
 - ruby-head
-- 2.2
-- 2.1
-- 2.0.0
```
Comment (Spaceghost) [inline .travis.yml]: Tests might pass, but is dropping these rubies the right thing? I'm fine with either way, but sneakers has less viability when it only cares to support the latest stable.

Comment (pbollenbeck) [inline .travis.yml]: The problem lies in the development dependencies. We could remove at least `metric_fu` which depends on `activesupport` and `rails_best_practices` which have pretty strict ruby requirements themselves. `metric_fu` doesn't seem to be used in the CI as far as I have gone through the code (but I'm not too familiar with it yet).
I'd in general favor a lighter list of dev dependencies anyway, IMO also `simplecov`, `simplecov-rcov-text`, `guard`, `guard-minitest`, `ruby-prof` and `metric_fu` could be removed without harm.

Comment (Spaceghost) [inline .travis.yml]: See #204, where I fixed that.

Comment (pbollenbeck) [inline .travis.yml]: @jondot any preference?

**Comment by michaelklishin** [comment]
👍

**Comment by michaelklishin** [comment]
I'll stick my neck out and merge this because having functional CI is very important. @jondot I hope requiring Ruby 2.1+ is reasonable. Some of the changes can be undone if you disagree.

**Comment by gabrieljoelc** [comment]
from https://github.com/jondot/sneakers#compatibility:
> * Sneakers 1.1.x and up (using the new generation Bunny 2.x) - Ruby 2.x.x
> * Sneakers 1.x.x and down - Ruby 1.9.x, 2.x.x

Do we even know what was specific to Ruby `2.0.0`?

**Comment by jondot** [comment]
There were some specific requirement for Ruby 1.9 but I believe there's no need to support these anymore. 2.x and up should be fine :thumbsup:

**Comment by jondot** [comment]
Also - this means 2.1.x and up is good to go!

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Sneakers
 
+[![Build Status](https://travis-ci.org/jondot/sneakers.svg?branch=master)](https://travis-ci.org/jondot/sneakers)
```
Comment (kruppel) [inline README.md]: @pbollenbeck Just in case is was unintentional, this is the second badge added to the README (the first is on line 23) 😄.

Comment (pbollenbeck) [inline README.md]: Oh, you're totally right. My apologies, I missed this. As the PR is already closed, how should we proceed? New PR?

**Pull request merged**